shared_model/backend/protobuf/impl/proto_query_response_factory.cpp

```diff
@@ -196,6 +196,32 @@ shared_model::proto::ProtoQueryResponseFactory::createTransactionsResponse(
       query_hash);
 }
 
+std::unique_ptr<shared_model::interface::QueryResponse>
+shared_model::proto::ProtoQueryResponseFactory::createTransactionsPageResponse(
+    std::vector<std::unique_ptr<shared_model::interface::Transaction>>
+        transactions,
+    const crypto::Hash &next_tx_hash,
+    interface::types::TransactionsNumberType all_transactions_size,
+    const crypto::Hash &query_hash) const {
+  return createQueryResponse(
+      [transactions = std::move(transactions),
+       &next_tx_hash,
+       &all_transactions_size](
```

Comment:
If `all_transactions_size` is a "base" class (I don't know, is it something like `uint_32`?), then there's no need to capture it by reference, by value it should be

Reply:
I think both options how to capture it is fine. It is only a matter of the code style